lisa/sut_orchestrator/azure/common.py

```diff
@@ -639,6 +638,16 @@ def delete_file_share(
     share_service_client.delete_share(file_share_name)
 
 
+def get_resource(resources: Any, type_name: str) -> Any:
```

Comment:
This method is not clear. The parameters and return are `Any`. And there is no explanation what it do. What kind of "resource" is it?

Reply:
update

Reply:
`find_by_name`, it's like the comments described.

Reply:
update